docs/quickstart.md

```diff
@@ -62,6 +62,22 @@ sudo apt-get install python3.7-dev
 
```

Comment:
actually, on Ubuntu LTS 20.04 the default version is 3.8, and on Ubuntu LTS 18.04 the default is 3.7. Maybe we could list the two cases.

Reply:
I mean why not just say do it for the version which is relevant to you (i.e. first check version, then run appropriate command)